src/Integration/LocalServices/IBindingChecker.cs

```diff
@@ -58,15 +54,7 @@ public async Task<bool> IsBindingUpdateRequired(CancellationToken token)
                 return true;
             }
 
-            var unboundProjects = unboundProjectFinder.GetUnboundProjects().ToArray();
-            var hasUnboundProjects = unboundProjects.Length > 0;
-
-            if (hasUnboundProjects)
-            {
-                logger.WriteLine(Strings.SonarLintFoundUnboundProjects, unboundProjects.Length, string.Join(", ", unboundProjects.Select(p => p.UniqueName)));
-            }
-
-            return hasUnboundProjects;
+            return false;
```

Comment:
Minimal change to keep tests passing.
I suspect this whole class will be deleted later on.